client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -412,26 +397,6 @@ export class PluginMeta extends Component {
 		}
 	}
 
-	isVersionCompatible() {
```

Comment:
Skimming over this component, I noticed you can also remove the unused `getPlan` method. What caught my attention is that it calls `selectedSite.plan`, but `.plan` is not a function.

Reply:
Ha - good catch! Removed in f7c321e431ba5d208f6d5136a5d1b5d2dfbc24fa